src/Configuration/Schema/SchemaConfigurationFactory.php

```diff
@@ -67,8 +67,8 @@ public function create(string $path, stdClass $rawConfig): SchemaConfiguration
             self::normalizeString($rawConfig->bootstrap ?? null),
             self::normalizeString($rawConfig->initialTestsPhpOptions ?? null),
             self::normalizeString($rawConfig->testFrameworkOptions ?? null),
-            self::normalizeString($rawConfig->xmlCoveragePath ?? null),
-            self::normalizeString($rawConfig->junitLogPath ?? null)
+            self::normalizeString($rawConfig->xmlCoverage ?? null),
+            self::normalizeString($rawConfig->logJunit ?? null)
```

Comment:
Variable names were on point tbh.

Reply:
@sanmai Sorry, I don't understand you. Would you be so kind to add more context here?

Reply:
The names like `xmlCoveragePath` you were using were excellent and on point.

Reply:
Changing variable here means that I need to change name of setting in configuration file, but I prefer to have the same parameters in CLI and configuration, so I prefer to leave it as is here.

Reply:
I can't agree with you here. CLI parameters are such because of PHPUnit. Here we're not limited by the same constraint.

Reply:
Waiting for review from @maks-rafalko

Reply:
I'd like to give this a green light, but this part is what I really like to see fixed.

Reply:
We need third opinion. I really don't like situation, when we have CLI options and configuration settings differently named. Problem it that you need to read more documentation and it could be additional point of failure, because you cannot easily transfer data from command line to config, because you have to alter it

Reply:
What you did here is really great, and I'd say with this small naming issue fixed it'll all will be just _perfect_.

Reply:
@sanmai another option is revert command line parameters name to initial in this PR.

Reply:
This is a truly despicable option tbh.

Reply:
@sanmai so, are you insisting on changing configuration file settings? In this case I'm going to rollback CLI options to their initial names because I want to have consistency between configuration and command line.

Reply:
I'd rather see both of these changes happen simultaneously. I don't understand what we're arguing about here. You can check other pull requests to see that purposeless nitpicking isn't something I find joy in. Neither I fancy spending the little free time I have arguing without arguments. ("I really don't like situation" isn't sound and neither an argument, but an appeal to a subjective taste in things.)

Therefore, this will be my final review of this PR unless changes are made, or there a _sound_ argument. Thank you.

Reply:
Difficult question ;) Let me share some thoughts/facts on this topic:

1. From what I see, we don't have any example when CLI option is named differently that corresponded config
2. we have many inconsistencies in **already existing** config settings and proposed settings in this PR (`logJunit` or `junitLogPath`

here is what I mean:

* `bootstrap` config settings - it's a path to a file, but it's not `bootstrapPath` (bad)
* `tmpDir` / `configDir` - this is a path to **directory**, hence `***Dir` postfix

What is proposed here by @sanmai:

* `xmlCoveragePath` for a **dir**
* `junitLogPath` for a **file**

As you can see, neither existing config in `master` branch, nor proposed names are correc. They are not consistent (`bootstrap` has no `Path` postfix, `xmlCoveragePath` should be `xmlCoverageDir`, and third issue - CLI options are named differently

What I can suggest: we must agree on the following:

1. if it's a path - postfix must be `***Path`
2. if it's a dir - postfix must be `***Dir`
3. what to do with different names of CLI options and config settings

Personally, 

* I find `xmlCoverageDir` and `junitLogPath` much more self-explaining thatn variants with `Path`/`Dir` postfix
* I find naming CLI options and config settings in a similar way - more clear than naming them differently. But if you both really like PHPUnit-like names (`coverage-xml`, `log-junit`) - let it be so.

I didn't get why don't you like the following approach:

* `--xml-coverage-dir` and `xmlCoverageDir` config setting
* `--junit-log-path` and `junitLogPath` config setting

One more thing to note: Infection is not only about PHPUnit, it's about PHPSpec and Codeception as well, which simply don't have `--coverage-xml` and `--log-junit` settings

Reply:
Let me give it a shot an explaining.  Take this textbook example on using Infection with PHPUnit at CI:

```
vendor/bin/phpunit --coverage-xml=build/logs/coverage-xml --log-junit=build/logs/junit.xml
vendor/bin/infection --coverage=build/logs ... (skipping other options for brevity)
```
Now, let's imagine we're using these new options, but named not as in PHPUnit:
```
vendor/bin/phpunit --coverage-xml=build/logs/coverage-xml --log-junit=build/logs/junit.xml
vendor/bin/infection --xml-coverage-dir=build/logs/coverage-xml --junit-log-path=build/logs/junit.xml
```
Well, you can kinda see what's going where and why. How likely you will remember these? I don't think you will, probably will need to Google up. Which means these two new options not very far away from the old single option, usability-wise.

But what if we'll be using these new options named as they are in PHPUnit:
```
vendor/bin/phpunit --coverage-xml=build/logs/coverage-xml --log-junit=build/logs/junit.xml
vendor/bin/infection --coverage-xml=build/logs/coverage-xml --log-junit=build/logs/junit.xml 
```
Boom! You already remember how to use them. You will probably write them out of your mind. That's a different kind of UI. 

Make sense?

Reply:
makes sense

Reply:
So, what should I do here?

Reply:
@maks-rafalko @sanmai What should I do with parameters and configuration file settings? Should I leave them in current state or change?
After finishing with this question, I'm going to proceed with documentation and tests.
Thank you.

Reply:
as for me, let's proceed with the current state: `log-junit` and `coverage-xml` names, same as for PHPUnit.

Other inconsistencies described above should/will be addressed in another PR